src/apps/vpn/qmake/platforms/ios.pri

```diff
@@ -66,6 +66,7 @@ OBJECTIVE_SOURCES += \
     apps/vpn/platforms/ios/iosglue.mm \
     apps/vpn/platforms/ios/iosnetworkwatcher.mm \
     apps/vpn/platforms/ios/iosnotificationhandler.mm \
+    apps/vpn/platforms/ios/iosupdater.mm \
```

Comment:
Please add `iosupdater.mm` and `iosupdater.h` to CMake as well.

Reply:
I did, right?